demisto_sdk/commands/validate/validators/PA_validators/PA114_pack_metadata_version_should_be_raised.py

```diff
@@ -0,0 +1,184 @@
+from __future__ import annotations
+
+import typing
+from typing import Iterable, Union
+
+from packaging.version import Version
+
+from demisto_sdk.commands.common.constants import (
+    PACK_METADATA_REQUIRE_RN_FIELDS,
+    SKIP_RELEASE_NOTES_FOR_TYPES,
+    FileType,
+)
+from demisto_sdk.commands.common.tools import find_type
+from demisto_sdk.commands.content_graph.objects import (
+    AssetsModelingRule,
+    BasePlaybook,
+    BaseScript,
+    CaseField,
+    CaseLayout,
+    CaseLayoutRule,
+    Classifier,
+    CorrelationRule,
+    Dashboard,
+    GenericDefinition,
+    GenericField,
+    GenericModule,
+    GenericType,
+    IncidentField,
+    IncidentType,
+    IndicatorField,
+    IndicatorType,
+    Integration,
+    Job,
+    Layout,
+    LayoutRule,
+    List,
+    Mapper,
+    ModelingRule,
+    ParsingRule,
+    Playbook,
+    PreProcessRule,
+    Report,
+    Script,
+    Trigger,
+    Widget,
+    Wizard,
+    XDRCTemplate,
+    XSIAMDashboard,
+    XSIAMReport,
+)
+from demisto_sdk.commands.content_graph.objects.pack import Pack
+from demisto_sdk.commands.validate.validators.base_validator import (
+    BaseValidator,
+    ValidationResult,
+)
+
+ContentTypes = Union[
+    Pack,
+    Integration,
+    Script,
+    Playbook,
+    AssetsModelingRule,
+    BasePlaybook,
+    BaseScript,
+    CaseField,
+    CaseLayout,
+    CaseLayoutRule,
+    Classifier,
+    CorrelationRule,
+    Dashboard,
+    GenericDefinition,
+    GenericField,
+    GenericModule,
+    GenericType,
+    IncidentField,
+    IncidentType,
+    IndicatorField,
+    IndicatorType,
+    Job,
+    Layout,
+    LayoutRule,
+    Mapper,
+    ModelingRule,
+    Pack,
+    ParsingRule,
+    Playbook,
+    PreProcessRule,
+    Report,
+    Script,
+    Trigger,
+    Widget,
+    Wizard,
+    XDRCTemplate,
+    XSIAMDashboard,
+    XSIAMReport,
+    List,
+]
+
+
+class PackMetadataVersionShouldBeRaisedValidator(BaseValidator[ContentTypes]):
+    error_code = "PA114"
+    description = "Ensure that the pack metadata version is raised on relevant changes."
+    rationale = (
+        "When updating a pack, its version needs to be raised to maintain traceability."
+    )
+    error_message = (
+        "The pack version (currently: {old_version}) needs to be raised - "
+        "make sure you are merged from master and "
+        'update the "currentVersion" field in the '
+        "pack_metadata.json or in case release notes are required run:\n"
+        "`demisto-sdk update-release-notes -g` - for automatically generation of release notes and version\n"
+        "`demisto-sdk update-release-notes -i Packs/{pack} -u "
+        "(major|minor|revision|documentation)` for a specific pack and version."
+    )
+
+    @staticmethod
+    def should_bump_is_metadata(content_item: ContentTypes):
```

Comment:
Keep in mind that if the content_item was collected it doesn't mean we need to bump.
For example - integration image change doesn't require version bump but it'll collect the integration object.

Reply:
In this case will the content_item.path point to the integration.yml or to the integration image?